Sources/Array+Remove.swift

```diff
@@ -35,7 +35,7 @@ extension Array {
 
      - parameter object: The object to be removed (if found)
      */
-    mutating func removeObject<T: Equatable>(_ object: T) {
+    public mutating func removeObject<T: Equatable>(_ object: T) {
```

Comment:
Nit pick: If you `import Swift` and cmd-click on the Swift and look at the implementation of `Array`, they use `Element` rather than `T` as the generic type name.